tests/phpunit/data/blocks/internationalized/block.json

```diff
@@ -0,0 +1,21 @@
+{
+	"name": "tests/internationalized",
+	"title": "This is a dummy plugin",
+	"description": "This is a dummy plugin",
+	"keywords": [
+		"alert",
+		"This is a dummy plugin"
+	],
+	"styles": [
+		{
+			"name": "default",
+			"label": "Default",
+			"isDefault": true
+		},
+		{
+			"name": "other",
+			"label": "This is a dummy plugin"
+		}
+	],
+	"textdomain": "internationalized"
+}
```

Comment:
I'd use different strings for each to make it more clear

Reply:
If I have to build another translation file, I will probably reuse the existing `block.json` that has those values unique.